.nxignore

```diff
@@ -1,2 +1,3 @@
 nx-dev/**/jest.config.js
 .next
+__fixtures__
```

Comment:
This is needed because some of our tests use `__fixtures__` that contain jest config files, and we do not want to create targets/projects from them.

Reply:
Hm we don't want to `.nxignore` them... I would add them to the `exclude` array along with `e2e`